llvm/include/llvm/Target/GenericOpcodes.td

```diff
@@ -110,6 +110,12 @@ def G_GLOBAL_VALUE : GenericInstruction {
   let hasSideEffects = false;
 }
 
+def G_PTRAUTH_GLOBAL_VALUE : GenericInstruction {
```

Comment:
Could you please add an entry to:
https://github.com/llvm/llvm-project/blob/main/llvm/docs/GlobalISel/GenericOpcode.rst

Reply:
Thanks for suggestion! Done